### ISSUE TITLE
moving AF to a new installation

### ISSUE THREAD

**Issue by shaieden** [ActiveForums/ActiveForums#264]
Wow, it's been a long time since I checked the forums of any DNN module, and so much has changed (for both the good and the bad!)

Anyway, I am trying to move an old forum of AF to a new fresh installation of dnn7+AF6.

Does anyone know how to do this? what should be done, which tables should be moved, which doesn't, what files, what settings... ? 

Thanks!
Shai Eden

**Comment by artfuld** [comment]
What version of AF do you have? AF has upgrade scripts from AF4.0

**Comment by shaieden** [comment]
I have ver 6 but i need to move the installation completely to a new installation of dnn. Its the upgrade I'm struggling with, but the migration...



‫ב-8 במאי 2015, בשעה 11:42, ‏‏Joakim Andersson ‏<notifications@github.com> כתב/ה:‬

**Comment by artfuld** [comment]
This was recently discussed in this thread #254 
If it's a fresh new installation of DNN your best move would probably be to install AF and then just copy all of the activeforums-tables from the old installation to the new. 
Make sure you update the old one to the latest update first.

Then you need to insert AF as a module with the same moduleId as it was on the old dnn-installation. And everything else revolving around the modules.

In short I would say that this is not worth doing, the chance of you missing some setting that is vital is rather big.

**Comment by supermansrf** [comment]
Not to mention what about users? If user ID's weren't exact aren't you going to have messed up post authors? I don't see where this is possible. I've been trying to figure out how to merge 2 sites into one and get stuck every time. Don't think it's going to be easily doable.

**Comment by shaieden** [comment]
Well, sometimes you just need to move to a fresh install… 

The users have been moved already successfully.

I wrote here a few steps which I think would be my path to the move:

1. install AF on the new DNN installation.

2. Clean the tables of the new AF.

3. Clean the old AF tables to not have any conflicts with Foreign Keys (Like userid which doesn’t exist anymore in the users table).

4. move the tables one by one with “allow identity insert”

5. move the files: images, avatars, attachments, themes.

6. Sync files on the new DNN.

How does that sound? Did I miss something?

**Comment by artfuld** [comment]
First you need to check that the users got the same ID as the did in the previous install.
But, if you can do all this and they will allow to get the identity inserted it should be fine, might need to change the moduleId but that should be a simple change.
